docs/topics/wasm/wasm-debugging.md

```diff
@@ -21,7 +21,7 @@ Create a project using the Kotlin Multiplatform wizard:
 3. Select the **Web** option. Make sure that no other options are selected.
 4. Click the **Download** button and unpack the resulting archive.
 
-![Kotlin Multiplatform wizard](wasm-compose-web-wizard.png){width=400}
+![Kotlin Multiplatform wizard](wasm-compose-web-wizard.png){width=450}
```

Comment:
```suggestion
   ![Kotlin Multiplatform wizard](wasm-compose-web-wizard.png){width=450}
```
Added indent so that the image is in line with the other steps.